src/utils.ts

```diff
@@ -1,10 +1,6 @@
-// @flow
-
 export function mapRequestInfoToUrlString(requestInfo: RequestInfo): string {
   if (requestInfo instanceof Request) {
     return requestInfo.url;
-  } else if (requestInfo instanceof URL) {
-    return requestInfo.href;
```

Comment:
What is this for?

Reply:
Because RequestInfo can't be `instanceof URL` I would guess.